### ISSUE TITLE
Image scaling doesn't update after refresh

### ISSUE THREAD

**Issue by briehanlombaard** [ericmandel/js9#19]
Hi Eric,

There was a change at some point that makes it so that an image's scaling doesn't get updated when I do a refresh which sometimes can produce a completely saturated image. 

I've tracked it down to [this commit](https://github.com/ericmandel/js9/commit/a292bf2868c02b038919100e3b3df8c388976de2) and the exact change in there is this bit right here:

    @@ -1472,12 +1478,10 @@ JS9.Image.prototype.mkRawDataFromHDU = function(obj, opts){
         // init the logical coordinate system, if possible
         this.initLCS(this.raw.header);
         // set initial scaling values if not done already
    -    if( isNaN(this.params.scalemin) ||
    -       (this.params.scaleclipping === "dataminmax") ){
    +    if( isNaN(this.params.scalemin) ){
            this.params.scalemin = this.raw.dmin;
         }
    -    if( isNaN(this.params.scalemax) ||
    -       (this.params.scaleclipping === "dataminmax") ){
    +    if( isNaN(this.params.scalemax) ){
            this.params.scalemax = this.raw.dmax;
         }
         // allow chaining

This is probably done for a reason, so I'm wondering if there's a way to get around it?

**Comment by ericmandel** [comment]
@briehanlombaard But not a good reason: this is the second mistake I've made while trying to squeeze too much performance out of the code. Those lines should be replaced by the generalized method:
````
  this.dataminmax();
````
and the 4 prior !isNaN(tval) checks that set the dmin and dmax should be removed. If you just call this.dataminmax() (and let me know that the problem is fixed), I'll clean up the code as soon as I have finished 3D cube support.
